src/interfaces/IERC20.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.21;
+pragma solidity >=0.5.0;
```

Comment:
Requested by Superform, this helps integrations use the interfaces without being locked to solidity 0.8.21